src/jq.js

```diff
@@ -23,7 +17,11 @@ export const run = (filter, json, options = {}) => {
     const { cmd, params } = createJqCommand(filter, json, options)
     execa(cmd, params)
     .then(({ stdout }) => {
-      return resolve(JSON.parse(stdout))
+      if (options.output === 'json') {
```

Comment:
I understand your point but I think it's good to have the option to modify the output... how do you think we should do it, if not here?